WcaOnRails/db/migrate/20220410022442_add_is_using_wca_live_for_scoretaking_to_competition.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class AddIsUsingWcaLiveForScoretakingToCompetition < ActiveRecord::Migration[6.0]
```

Comment:
Rename migration in conjunction with the field name itself, see my other comment above.